shared/@hedvig-ui/Table/table.tsx

```diff
@@ -168,10 +170,10 @@ export const TableRow: React.FC<TableRowProps> = ({
             onResolve(index)
           },
           neighbors: {
-            up: index > 0 ? `Table Row ${index - 1}` : topElement,
+            up: index > 0 ? `Table Row - ${index - 1}` : topElement,
             down:
               index < length - 1
-                ? `Table Row ${index + 1}`
+                ? `Table Row - ${index + 1}`
```

Comment:
Also if we're gonna be consistent this should maybe even be:
```suggestion
                ? `TableRow - ${index + 1}`
```

Or we should split up `UserProfile` in the other PR